src/store/Store.js

```diff
@@ -71,9 +71,14 @@ class Store {
       {
         type: DISPATCH_TYPE,
         payload: data
-      }, ({error, value}) => {
+      }, (resp) => {
+        const {error, value} = resp;
+
         if (error) {
-          reject(assignIn((new Error()), error));
+          let errMsg = "\nLooks like there is an error in the background page. " +
```

Comment:
I will be introducing linting sometime soon to this repo. For multi-line messages like this, could we use templates where possible and `const` when possible. I think we should hoist the beginning of this error in a `const` variable and get rid of the triple-nested call (`Error` constructor in `assignIn` in `reject` - I know this existed beforehand - just pointing it out).

```
const backgroundErrPrefix = '\nLooks like there is an error in the background page. ' +
  'You might want to inspect your background page for more details.\n';
```

Then:

```
if (error) {
  const bgErr = new Error(`${backgroundErrPrefix}${error}`);
  reject(assignIn(bgErr, error));
}
```

I like the friendliness of the error message itself though. 🎉